<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QDialog" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new timer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_reverse">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_continue_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
